CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -442,8 +442,11 @@ public static function getTemplateContribution($id, $overrides = []) {
     }
     if ($templateContributions->count()) {
       $templateContribution = $templateContributions->first();
-      $result = array_merge($templateContribution, $overrides);
       $lineItems = CRM_Price_BAO_LineItem::getLineItemsByContributionID($templateContribution['id']);
+      if (count($lineItems) > 1 && isset($overrides['financial_type_id'])) {
```

Comment:
This could use a short comment to explain why

Reply:
done